packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/Server/index.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Collapsible, Text, Element } from '@codesandbox/components';
+import { Tasks } from './Tasks';
+import { Status } from './Status';
+import { Ports } from './Ports';
+import { Control } from './Control';
+import { EnvVars } from './EnvVars';
+
+export const Server = () => (
+  <Collapsible defaultOpen title="Server Control Panel">
+    <Element padding={2} marginBottom={5}>
+      <Text weight="medium" block marginBottom={2}>
+        Server Sandbox
+      </Text>
+      <Text variant="muted">
+        This sandbox is executed on a server. You can control the server from
+        this panel.
+      </Text>
+      <Element marginTop={5}>
```

Comment:
I would stick to marginBottom unless absolutely necessary

Reply:
Changed :)

Reply:
Err not sure if I'm looking at an outdated version, but I see marginTop?